packages/inputs/src/Table.js

```diff
@@ -208,6 +208,23 @@ export class Table extends MosaicClient {
     return this;
   }
 
+  activate() {
```

Comment:
What's the logic here? I see this calling selection update repeatedly for EVERY row and that does not look right... I would expect `selection.activate` (not `update`) to be called just once.